File: app/src/main/java/com/pietrantuono/podcasts/singlepodcast/model/SimpleEnclosure.java
package com.pietrantuono.podcasts.singlepodcast.model;

import com.rometools.rome.feed.CopyFrom;
import com.rometools.rome.feed.synd.SyndEnclosure;

import io.realm.RealmObject;

public class SimpleEnclosure extends RealmObject implements SyndEnclosure  {
    private int unused;

    public SimpleEnclosure(SyndEnclosure syndEnclosure) {

    }

    public SimpleEnclosure() {
    }

    @Override
    public String getUrl() {
        return null;
    }

    @Override
    public void setUrl(String url) {

    }

    @Override
    public long getLength() {
        return 0;
    }

    @Override
    public void setLength(long length) {

    }

    @Override
    public String getType() {
        return null;
    }

    @Override
    public void setType(String type) {

    }

    @Override
    public Class<? extends CopyFrom> getInterface() {
        return null;
    }

    @Override
    public void copyFrom(CopyFrom obj) {

    }
}

File: data/src/main/java/pojos/SimpleEnclosure.kt
package pojos

import com.rometools.rome.feed.CopyFrom
import com.rometools.rome.feed.synd.SyndEnclosure
import io.realm.RealmObject

open class SimpleEnclosure : RealmObject, SyndEnclosure {
    private var url: String? = null
    private var length: Long = 0
    private var type: String? = null

    constructor(syndEnclosure: SyndEnclosure) {
        setLength(syndEnclosure.length)
        setUrl(syndEnclosure.url)
        setType(syndEnclosure.type)
    }

    constructor() {}

    override fun getUrl(): String? {
        return url
    }

    override fun setUrl(url: String) {
        this.url = url
    }

    override fun getLength(): Long {
        return length
    }

    override fun setLength(length: Long) {
        this.length = length
    }

    override fun getType(): String? {
        return type
    }

    override fun setType(type: String) {
        this.type = type
    }

    override fun getInterface(): Class<out CopyFrom>? {
        return null
    }

    override fun copyFrom(obj: CopyFrom) {

    }
}